src/js/binder.js

```diff
@@ -0,0 +1,128 @@
+/*
+
+    Add persistent bindings between a selector and a model value.  Changes are propagated between the two. See the
+    documentation for more details:
+
+    https://github.com/GPII/gpii-binder/
+
+    This code was originally written by Antranig Basman <amb26@ponder.org.uk> and with his advice was updated and
+    extended by Tony Atkins <tony@raisingthefloor.org>.
+
+*/
+/* global fluid, jQuery */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.binder");
+
+    /**
+     *
+     * Try to evolve a given string into a typed variable or object.
```

Comment:
This is a pretty thin description of quite a complex algorithm. Can you elaborate what it does exactly, and in what context? The word "evolve" doesn't convey very much. I think the workflow of trying to parse as JSON and then doing nothing in the case of failure is inherently suspicious (as we've remarked elsewhere) - can you think of a way of reforming the algorithm such that it is sure what types it will be operating on?